### ISSUE TITLE
File sent as list of JSON's causes logging failure

### ISSUE THREAD

**Issue by FreddieMatherSmartDCSIT** [xtuml/munin#154]
If a list of events as JSON objects is sent to the Protocol Verifier Kafka AER Reception topic (using the correct payload structure) the PV will fail and the Reception log file will print a long series of never ending `]` within the file. This causes the log file to breach the log rotation size and files of up to 8GB have been seen.

Steps to reproduce:

- Use a kafka producer to send a list of JSON (events) to the AEReception ingestion topic
- Check the size of the Reception log file
- tail the log file to see a large out put of `]`

**Comment by cortlandstarrett** [comment]
Do I understand that you attempted to send an array of events to the topic rather then sending them one at a time?

**Comment by FreddieMatherSmartDCSIT** [comment]
Yeah sorry my python brain has me talking about lists! Yes we sent an array of events and observed that peculiar behaviour in the logs

**Comment by FreddieMatherSmartDCSIT** [comment]
Yes we prepended the string length to the array.

**Comment by cortlandstarrett** [comment]
We plan to add support for receiving an array of audit events.  The restriction will be that the array must contain events for a single job (all events having the same JobID).  If the events are already ordered, the performance of the PV will benefit.

**Comment by cortlandstarrett** [comment]
Support arrays of events (complete jobs or otherwise) is now in place.
These are in main as of 11 Feb 2025.
They will be present in the next release (end of March).
